src/Microsoft.Health.Fhir.Liquid.Converter/DotLiquids/MergeDiff.cs

```diff
@@ -0,0 +1,135 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using DotLiquid;
+using DotLiquid.Exceptions;
+using DotLiquid.Util;
+using Microsoft.Health.Fhir.Liquid.Converter.Models;
+using Newtonsoft.Json;
+using RenderException = Microsoft.Health.Fhir.Liquid.Converter.Exceptions.RenderException;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.DotLiquids
+{
+    public class MergeDiff : Block
+    {
+        private static readonly Regex Syntax = R.B(@"^({0}+)\s$", DotLiquid.Liquid.VariableSignature);
```

Comment:
I tested in Windows platform and this regex can't catch the `msg` markup correctly, while this one is fine to catch: `@"({0}+)"`.

Reply:
Wired thing is that after I reopened the solution everything works well again, so just ignore this comment.